File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/java/non-vul/code_before16.java
// From cwe-snippets, snippets_90/non-compliant/Java/cwe-0511/rand_01.java

private void good1() throws Throwable
{
    if ((new SecureRandom()).nextInt() == 20000)
    {
        IO.writeLine("Sorry, your license has expired.  Please contact support.");
    }
}

File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/cpp/non-vul/code_before16.cpp
// From cwe-snippets, snippets_90/non-compliant/C++/cwe-0762/delete_array_char_malloc_01.cpp

static void func()
{
    char * data;
    /* Initialize data*/
    data = NULL;
    data = (char *)malloc(100*sizeof(char));
    if (data == NULL) {exit(-1);}
    free(data);
}

